crawler.py

```diff
@@ -40,11 +40,15 @@
 
 RESTART_RETRIES = 5
 
+extract = TLDExtract()
```

Comment:
We should move this back inside a function that runs only when we run a crawl, because we only need this object when running a crawl. Initializing `TLDExtract` fetches the PSL. No need to fetch the PSL (among other things) when, say,  we're just printing usage.

Reply:
Should we disable [caching](https://github.com/john-kurkowski/tldextract#note-about-caching)?

Reply:
Good thinking, reworked this so that declaring `extract` now takes place in the two methods that use it